generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -409,23 +409,24 @@ _%>
             .setValidator(validator).build();
     }
 
+<%_ ['DEFAULT_', 'UPDATED_'].forEach((fieldStatus) => { _%>
     /**
-     * Create an entity for this test.
+     * Create an <% if (fieldStatus === 'UPDATED_') { %>updated<% } %> entity for this test.
```

Comment:
Should be: `Create an <% if (fieldStatus === 'UPDATED_') { %>updated <% } %>entity for this test.`
Otherwise, we can have: `* Create an  entity for this test.`, with 2 spaces between `an` and `entity`